### PR TITLE
[SDE-2272] Add eu-south-2 and associated AMIs and instance types

### PR DESCRIPTION
**Pull request by dustman9000**
# What is being added?
Add eu-south-2 and associated AMIs and instance types

Ref: [SDE-2272](https://issues.redhat.com//browse/SDE-2272)

### CODE REVIEW

File: hack/templates/aws.managed.openshift.io_v1alpha1_configmap.tmpl [hunk 1]
```diff
@@ -41,6 +41,7 @@ objects:
       eu-west-2:ami-0188c0c5eddd2d032:t2.micro
       eu-west-3:ami-0c4224e392ec4e440:t2.micro
       eu-south-1:ami-04684e5a51afd7579:t3.micro
+      eu-south-1:ami-0ff2e412ffb1b6aac:t3.micro
```
Comment (AlexVulaj) [inline hack/templates/aws.managed.openshift.io_v1alpha1_configmap.tmpl:44]: Is this a typo?

Comment (dustman9000) [reply hack/templates/aws.managed.openshift.io_v1alpha1_configmap.tmpl]: fixed!

**Review by iamkirkbater — APPROVED** [review]
/approve

**Pull request merged**